igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -15,10 +15,11 @@
     "\n",
     "In this notebook we will:\n",
     "\n",
+    "<br></br>",
     "1. Use the [HyP3 Python SDK](https://hyp3-docs.asf.alaska.edu/using/sdk/) to:\n",
     "   - Request On Demand InSAR products from ASF HyP3\n",
     "   - Download the InSAR products when they are done processing\n",
-    "<br></br>\n",
+    "<br></br>",
```

Comment:
When this break was added, it also adjusted the indentation level so that it was no longer aligned with the first numeric. I added another break before the first numeric, and that evens things up in the GitHub rendering; not sure if it will also behave as expected in OSL; the PyCharm rendering does weird things with it.